framework/core/src/Api/Serializer/ForumSerializer.php

```diff
@@ -135,4 +135,12 @@ public function getAssetUrl($assetPath): string
     {
         return $this->assetsFilesystem->url($assetPath);
     }
+
+    /**
+     * @return \Tobscure\JsonApi\Relationship|null
+     */
+    protected function actor($model)
```

Comment:
Could this be typed directly?

```suggestion
    protected function actor($model): ?\Tobscure\JsonApi\Relationship
```

Reply:
Something is telling me no for PHP 7.3? Bu then again it's really early and my mind is not awake..

Reply:
it's valid 7.3 syntax, we should import the class though instead of using the full qualified name directly